libmscore/layout.cpp

```diff
@@ -3314,8 +3314,10 @@ void alignHarmonies(const System* system, const std::vector<Segment*>& sl, bool
       QMap<int, QList<Element*>> staves;
       for (const Segment* s : sl) {
             for (Element* e : s->annotations()) {
-                  if ((harmony && e->isHarmony()) || (!harmony && e->isFretDiagram()))
-                        staves[e->staffIdx()].append(e);
+                  if ((harmony && e->isHarmony()) || (!harmony && e->isFretDiagram())) {
+                        if (e->autoplace() && e->isStyled(Pid::OFFSET))
```

Comment:
How does this change relate to the chord playback?

Reply:
Ooops, it doesn't, I was helping test something else about layout and I guess I didn't notice this was still there.  I'll update.

We do still need to resolve the question of how to handle the default, and whether we special case old scores, force playback sound to piano, or any of the other things we've discussed.  But first step is removing the preference pretty much no matter what.